package/upgrade/upgrade_node.sh

```diff
@@ -443,19 +445,22 @@ upgrade_os() {
   tmp_rootfs_mount=$(mktemp -d -p $HOST_DIR/tmp)
   mount $tmp_rootfs_squashfs $tmp_rootfs_mount
 
+  # replace the fixed elemental CLI for fix elemental upgrade issues
+  new_elemental_cli=$SCRIPT_DIR/$ELEMENTAL_DIR/elemental
+  target_elemental_cli=$HOST_DIR/usr/bin/elemental
   elemental_upgrade_log="${UPGRADE_TMP_DIR#"$HOST_DIR"}/elemental-upgrade-$(date +%Y%m%d%H%M%S).log"
   local ret=0
+  mount --bind $new_elemental_cli $target_elemental_cli
   chroot $HOST_DIR elemental upgrade --logfile "$elemental_upgrade_log" --directory ${tmp_rootfs_mount#"$HOST_DIR"} || ret=$?
   if [ "$ret" != 0 ]; then
     echo "elemental upgrade failed with return code: $ret"
     cat "$HOST_DIR$elemental_upgrade_log"
     exit "$ret"
   fi
+  umount $target_elemental_cli
```

Comment:
need to umount in the trap function `clean_up_tmp_files` or perhaps copying the file to a temporary place.

Reply:
NVM, it's already handled.